### ISSUE TITLE
Not all "hotness" tasks seem to be submitted in the `deliver_hotness` CRON task

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-server#14981]
It does not seem to affect -dev/-stage but it was very obvious in -prod after yesterday's push. We see logs for `reset` sub-tasks but nothing for `update` sub-tasks. Theoretically, there should be 1K `update` tasks and very few `reset` ones.

I don't see any errors in Sentry and the BQ call seems fine because the `reset` sub-tasks looked good (and we need the BQ data to prepare and submit those tasks).

**Comment by willdurand** [comment]
We found the root cause today and we were able to unlock the CRON task.

**Comment by willdurand** [comment]
> We found the root cause today and we were able to unlock the CRON task.

We found that one `FrozenAddon` had a GUID set to `None`, because this add-on was deleted and we apparently don't keep the list of frozen add-ons in sync (see: https://github.com/mozilla/addons-server/issues/15004).

We also found an error related to a BigQuery `addon_id` (GUID) with an invalid case like `something@moz.Org` instead of `something@moz.org`. MySQL is happy to return results but then the dict lookup fails (`averages`).
